pynest/examples/gap_junctions_two_neurons.py

```diff
@@ -70,15 +70,13 @@
 
 nest.Simulate(351.)
 
-senders = nest.GetStatus(vm, 'events')[0]['senders']
-times = nest.GetStatus(vm, 'events')[0]['times']
-V = nest.GetStatus(vm, 'events')[0]['V_m']
+senders = vm.events['senders']
+times = vm.events['times']
+v_m_values = vm.events['V_m']
 
 plt.figure(1)
-plt.plot(times[numpy.where(senders == 1)],
-         V[numpy.where(senders == 1)], 'r-')
-plt.plot(times[numpy.where(senders == 2)],
-         V[numpy.where(senders == 2)], 'g-')
+plt.plot(times[numpy.where(senders == 1)], v_m_values[numpy.where(senders == 1)], 'r-')
```

Comment:
Replace inefficient duplicated `where` calls.

Reply:
this is not directly related to nest-3 features, so we'll take this in another PR for this particular example.